lib/mintel/dashboards/components/panels/capacity.libsonnet

```diff
@@ -0,0 +1,312 @@
+local commonPanels = import 'components/panels/common.libsonnet';
+local statusdotsPanel = commonPanels.statusdots;
+local promQuery = import 'components/prom_query.libsonnet';
+local seriesOverrides = import 'components/series_overrides.libsonnet';
+{
+
+  cpuCoresRequests(nodeSelectorRegex, startRow, span=2)::
```

Comment:
Should the name specify the Widget type ? in this case a Gauge

Reply:
Should be good now.